main/src/library/Concurrent/Channel.flix

```diff
@@ -95,6 +95,8 @@ namespace Concurrent/Channel {
     ///   `waitingGetters` - a set of conditions that are waiting for an
     ///       element. This set is notified and cleared after each new element.
     ///       An available element is not guaranteed when notified.
+    ///   `rendezvous` - a condition that is waiting for a rendezvous (only
```

Comment:
As a reader, this almost cyclic definition is a bit confusing. Maybe there's a less abstract name than rendezvous? Can you expand a bit on this?